backblazeUpload.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/kothar/go-backblaze.v0"
+)
+
+func uploadFileToBackblaze(filename string, accountID string, applicationKey string, bucketName string) (string, error) {
```

Comment:
If this function will be exported, capitalize first letter and write comment ("UploadFileToBackblaze...").